cypress/src/core/action.js

```diff
@@ -4,6 +4,12 @@ export function setField(selector, inputText) {
 export function clickField(selector) {
     cy.get(selector).click()
 }
+export function checkField(selector) {
+    cy.get(selector).check()
```

Comment:
Maybe these function should be a space blank each other

Reply:
I agree with you.